src/Core/Framework/Store/Services/ExtensionLoader.php

```diff
@@ -246,6 +247,8 @@ private function loadLocalAppsCollection(Context $context): ExtensionCollection
                 'installedAt' => null,
                 'active' => false,
                 'type' => ExtensionStruct::EXTENSION_TYPE_APP,
+                'allowUpdate' => $app->isManagedByComposer(),
```

Comment:
shouldn't this be `!$app->isManagedByComposer()`? (negation)

Reply:
yes already fixed and added tests ;)

Reply:
never mind, it was updated ...